serverless/pages/cloud-regions.mdx

```diff
@@ -0,0 +1,26 @@
+---
+slug: /serverless/regions
+title: Serverless regions
+description: Index, search, and manage ((es)) data in your preferred language.
+tags: ["serverless", "regions", "aws", "cloud"]
+---
+
+A region is the geographic area where the data center of the cloud provider that hosts your project is located. Review the available Elastic Cloud Serverless regions to decide which region to use. If you aren't sure which region to pick, choose one that is geographically close to you to reduce latency. 
+
+Elastic Cloud Serverless handles all hosting details for you, and doesn't require any additional accounts with the underlying cloud provider. You are unable to change the region after you create a project. 
```

Comment:
I will just remove this sentence

, and doesn't require any additional accounts with the underlying cloud provider